clang/test/CodeGen/aarch64-sme-intrinsics/acle_sme_ldr.c

```diff
@@ -8,7 +8,7 @@
 // CHECK-C-LABEL: @test_svldr_vnum_za(
 // CHECK-CXX-LABEL: @_Z18test_svldr_vnum_zajPKv(
 // CHECK-NEXT:  entry:
-// CHECK-NEXT:    tail call void @llvm.aarch64.sme.ldr(i32 [[SLICE_BASE:%.*]], ptr [[PTR:%.*]])
```

Comment:
Were these changes to SLICE_BASE and PTR intentional? When I try to run this test now it fails with the following:
`undefined variable: SLICE_BASE` &
`undefined variable: PTR`

Reply:
These changes actually came from the test update script llvm/utils/update_cc_test_checks.py, but I can revert them as they weren't intentional changes.